synapse/push/bulk_push_rule_evaluator.py

```diff
@@ -87,8 +87,11 @@ def action_for_event_by_user(self, event, context):
         condition_cache = {}
 
         for uid, rules in self.rules_by_user.items():
-            display_name = room_members.get(uid, {}).get("display_name", None)
-            if not display_name:
+            display_name = None
+            profile_info = room_members.get(uid, {})
```

Comment:
The `{}` is confusing me because it doesn't make sense as a default value here.
However I see that the code works because of the `if profile_info` below.
Maybe `room_members.get(uid, None)` or just `room_members.get(uid)` would make more sense here?